src/expectRevert.js

```diff
@@ -1,14 +1,15 @@
 const { web3 } = require('./setup');
 
-const { expect } = require('chai');
 const colors = require('ansi-colors');
 const semver = require('semver');
 
 async function expectException (promise, expectedError) {
   try {
     await promise;
   } catch (error) {
-    expect(error.message).to.include(expectedError, `Wrong failure type, expected '${expectedError}'`);
+    if (error.message.indexOf(expectedError) === -1) {
+      throw Error(`Wrong failure type, expected '${expectedError}' and got '${error.message}'`);
```

Comment:
```suggestion
      throw new Error(`Wrong failure type, expected '${expectedError}' and got '${error.message}'`);
```

These two things are supposed to be the same but this is the more common way I think.

Reply:
According to https://es5.github.io/#x15.11.1,

>Thus the function call Error(…) is equivalent to the object creation expression new Error(…) with the same arguments.

We already have a `throw Error` in this file, and I don't want to change both :laughing: